src/www/ui/api/documentation/openapi.yaml

```diff
@@ -2289,6 +2327,81 @@ components:
             example: 
             - "path/to/file1"
             - "path/to/file2"
+    JobQueue:
+      description: Structure of jobs queue
+      type: object
+      properties:
+        jq_pk:
+          description: Job queue id
+          type: string
+        jq_job_fk:
+          description: Job id
+          type: string
+        jq_type:
+          description: Job queue type
+          type: string
+        jq_args:
+          description: Any arguements
+          type: string
+        jq_starttime:
+          description: Job queue start time
+          type: string
+        jq_endtime:
+          description: Job queue end time
+          type: string
+        jq_endtext:
+          description: Job queue completion status
+          type: string
+        jq_itemsprocessed:
+          description: Number of items processed
+          type: string
+        jdep_jq_depends_fk:
+          description: Job queue dependency id
+          type: string
```

Comment:
Can you please explain how `jq_job_fk`, `jq_type`, `jq_args` be helpful to the end user for this endpoint?

Reply:
Yes sure! 
- `jq_type` is the agent name that is scheduled (_eg - nomos, spdx2, ojo, monk, etc._), which is a parameter that will let the end user know about the kind of job it is. 
- `jq_args` gives us the `uploadId`. So, It might be significant in some places where we just want to use the `jobQueue` parameter for a particular job as it will let us know about the upload it is associated with. Same goes for `jq_job_fk`. Else we will have backtrack and get the `jobId` and `uploadId`.

Reply:
[UPDATE] Removed `jq_job_fk ` and `jq_args `.